MANIFEST.in

```diff
@@ -35,18 +35,19 @@ exclude .dockerignore
 exclude test_postgresql.sh
 exclude .editorconfig
 exclude sytest-blacklist
+exclude mypy.ini
+exclude .codecov.yml
+exclude .coveragerc
 
 include pyproject.toml
 recursive-include changelog.d *
 
 prune .buildkite
 prune .circleci
-prune .codecov.yml
-prune .coveragerc
 prune .github
 prune debian
 prune demo/etc
 prune docker
-prune mypy.ini
 prune snap
 prune stubs
+prune contrib
```

Comment:
So from what [I can see](https://packaging.python.org/guides/using-manifest-in/), `exclude` is "Exclude all files matching any of the listed patterns" and `prune` is "Exclude all files under directories matching dir".

Do we want to keep `contrib` without any items under it (actually why do we do this for the other directories)?

Reply:
According to what @richvdh said in #synapse-dev, I don't think so.

![image](https://user-images.githubusercontent.com/5547783/80968765-d0b36b00-8e18-11ea-8fbf-bfd3922d2a2d.png)

Reply:
Change it to an `exclude` rule then?

Reply:
CI seems to fail again with `exclude`, changing it back to `prune` to fix the CI and I'll look at making `prune`s `exclude`s in a later PR.